### ISSUE TITLE
Console Errors latest dev build

### ISSUE THREAD

**Issue by TenReeez** [BG-Software-LLC/WildStacker#142]
Running on 1.13.2 Paper

Dev build 106

Constant spamming in console of the following 2 errors Not sure if second one is related.
The errors mention slimes and turtles. Only had errors since using latest wildstaker devbuild and epicspawners 7.0.7    

When I first installed plugin I monitored it for a few hours no problem, But upon reboot issues started. 

https://pastebin.com/mzMhaz7q

Thank you

**Comment by OmerBenGera** [comment]
How is it related to the plugin?
It seems like mobs are spawning with same ids, which has nothing to do with the plugin.
Also, 1.13.2 is not a stable version, not sure why are you even using it...

**Comment by TenReeez** [comment]
NON RELATED TO WILDSTACKER

**Comment by TenReeez** [comment]
I have reopened the issue. 

When I place a turtle spawner via /stacker give 
The following error occurs on placement of spawner:
https://pastebin.com/hXy3H3yY

I dont use any other spawner plugin or stacker

**Comment by OmerBenGera** [comment]
The error seems to also occur when spawning a turtle using a spawn egg.
Therefore, as I said earlier, it's an issue with the spigot version itself, and not with WildStacker.
(Try spawning a turtle using a spawn egg)

**Comment by TenReeez** [comment]
Yes you are correct error occurs when i try to spawn with egg.  Thank you.

**Comment by TenReeez** [comment]
Hey sorry to be a pain.  

But the issue still persists, I have created a test server. Same plugins. The issue ONLY happens when Wildstacker is used.   Once Wildstacker is disabled, I can spawn in turtles with no error using spawn eggs. However once Wildstacker is running,  Turtle spawn eggs do not spawn turtles and the error as shown is created in console.

To highlight the issue, I created a server where the ONLY plugin being used is WIldstacker and the issue is replicated.   

1.13.2 Paper server

Thank you
